cdf-core/cdf/js-modules/queries/BaseQuery.js

```diff
@@ -205,6 +310,13 @@
       }
     },
 
+   /**
+    * Alias for {{#crossLink "BaseQuery/reprocessLastResults:method"}}reprocessLastResults{{/crossLink}}
+    *
+    * @method reprocessResults
+    * @param outsideCallback Success callback
+    * @throws  NoCachedResults error if there haven't been previous calls to the server
```

Comment:
missing returns tag
if this is just an alias, which it seems it is, wouldn't it be preferable to just call reprocessLastResults inside the function?